### PR TITLE
Update granulate-utils (CRI v1, resolve_proc_root_links fix, ...)

### PR DESCRIPTION
**Pull request by netaneld122**
Main updates:
1. Breaking change in gLogger SDK (Need to pass `AuthToken`
2. Add support for CRIv1 grpc
3. Fixes to resolve_proc_root_links

### CODE REVIEW

**Comment by Jongy** [comment]
Tests: https://github.com/Granulate/granulate-utils/pull/221#pullrequestreview-1796577677

**Comment by Jongy** [comment]
Once we merge it, it's also for https://github.com/Granulate/granulate-utils/pull/209

**Review by Jongy — APPROVED** [review]
LGTM, merge after we merge the granulate-utils one.

As for the title - just specify "Upgrade granulate-utils", it includes several things (also the CRI v1 support, etc, so no need to mention only resolve_proc_root_links).

File: gprofiler/log.py
Comment (michelhe) [inline gprofiler/log.py:1]: Pfft, granulate-utils wasn't update for this long? lol

**Review by Jongy — APPROVED** [review]
I hope it won't break things like containers listing, we had some changes in granulate-utils in the APIs (which we don't have adequate tests for). I'll give it a try after merging.

**Comment by netaneld122** [comment]
@Jongy should I bump the version?

**Comment by Jongy** [comment]
Nope @netaneld122

**Pull request merged**
